styles/_shop-item.scss

```diff
@@ -0,0 +1,32 @@
+.shop-item {
+  display: flex;
+  flex-direction: column;
+
+  max-width: $max-content-width;
+  border: $brand-border;
+
+}
+
+.shop-item__shirt {
+  margin: $gutter;
+}
+
+.shop-item__pic {
+  width: 100%;
+}
+
+.shop-item__info {
+  display: flex;
+  flex-direction: column;
```

Comment:
This is not required, the contents of the `__info` area is already stacked. Putting flex here is doubling up work and can lead to unexpected results